src/links.css

```diff
@@ -35,3 +35,17 @@
 .link.is-active {
   color: var(--default-primary-interactive-color-dark);
 }
+
+/**
+ * When a form element like `button` includes both the `link` class and the `disabled` attribute, it will be styled accordingly.
+ * The disabled property has no effect on `a` elements.
+ *
+ * @memberof Links
+ * @example
+ * <button href='#Links' disabled class='link'>A disabled link</button>
+ * <button href='#Links' disabled class='link link--red'>A disabled red link</button>
+ */
+.link:disabled {
+  cursor: default;
+  color: var(--inactive-text-color) !important;
```

Comment:
As mentioned in https://github.com/mapbox/assembly/issues/860#issuecomment-336714126, I wonder if there's a way to avoid `!important` here ...

Reply:
You okay with merging this PR, which follows the existing pattern, then make the fix in a bigger, new PR dedicated to purging assembly of !importants?

Reply:
Sure ✌️